ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/eea/EeaSendRawTransactionTest.java

```diff
@@ -143,7 +143,8 @@ public void invalidTransactionIsNotSentToEnclaveAndIsNotAddedToTransactionPool()
 
     final JsonRpcResponse expectedResponse =
         new JsonRpcErrorResponse(
-            validPrivateForTransactionRequest.getRequest().getId(), JsonRpcError.INVALID_PARAMS);
+            validPrivateForTransactionRequest.getRequest().getId(),
+            JsonRpcError.PRIVATE_TRANSACTION_FAILED);
```

Comment:
This should probably called PRIVATE_TRANSACTION_INVALID, because it failed during validation. Transactions fail when they are actually executed.

Reply:
added a new code for that